### PR TITLE
Add defaults to fixType element

### PR DESCRIPTION
**Pull request by redhatrises**
- See https://scap.nist.gov/specifications/xccdf/xccdf_element_dictionary.html#fixType

### CODE REVIEW

File: shared/utils/combine-remediations.py [hunk 1]
```diff
@@ -423,21 +423,21 @@ def main():
                         else:
                             mod_file += line
 
-                complexity = None
-                disruption = None
-                reboot = None
+                complexity = "unknown"
+                disruption = "unknown"
+                reboot = "false"
```
Comment (mpreisler) [inline shared/utils/combine-remediations.py]: is this a safe default? saying that fix doesn't require reboot when we are not sure doesn't seem right.

Comment (redhatrises) [inline shared/utils/combine-remediations.py]: @mpreisler the defaults are based off of the specification which says that the default is '0'.

**Comment by mpreisler** [comment]
I quite like the semantic of us omitting defaults, makes the files smaller, doesn't mislead the user. Is there any value in listing strategy as "unknown" if it's not specified?

Furthermore the reboot default is problematic IMO. It is in the spec but it can be very misleading.

**Comment by redhatrises** [comment]
I was thinking that it would be a good way to handle the scripts that don't have the content in them as well as being able to visually compare differences as there are scripts that don't match.

**Comment by mpreisler** [comment]
I am leaning NACK here. It's correct as far as spec goes but I like that we don't show `strategy` in the HTML report if it's unknown. I think that improves UX and is less confusing. I really dislike the `reboot` default, that may be very misleading.

Any thoughts from the rest of the community?

**Comment by redhatrises** [comment]
Closing. We can add this back in if requested for in the future.

**Pull request closed without merging**